### ISSUE TITLE
Sound information for critical alerts

### ISSUE THREAD

**Issue by bio** [ZhukV/AppleApnPush#65]
Is it possible to extend Aps class to allow to set sound dictionary for a critical alert?
Currently the `$sound` attribute is private and has type `string`.

Keys to include in the sound dictionary

Key | Value Type | Description
-- | -- | --
critical | Number | The critical alert flag. Set to 1 to enable the critical alert.
name | String | The name of a sound file in your app’s main bundle or in the Library/Sounds folder of your app’s container directory. Specify the string "default" to play the system sound. For information about how to prepare sounds, see UNNotificationSound.
volume | Number | The volume for the critical alert’s sound. Set this to a value between 0.0 (silent) and 1.0 (full volume).

https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification

Thanks!

**Comment by ZhukV** [comment]
Yes, it's possible ;)

Please update library to `v3.0.10`. This feature was be merged in #66. After, you can use `\Apple\ApnPush\Model\Sound` object.

```php
$aps = new Aps(new Alert());
$aps = $aps->withSound(new Sound('sound-file', 0.5, true));
```

**Comment by bio** [comment]
Awesome! Thank you!
